src/text-editor-element.coffee

```diff
@@ -153,8 +153,10 @@ class TextEditorElement extends HTMLElement
     @component.focused() if event.relatedTarget is this
 
   addGrammarScopeAttribute: ->
```

Comment:
I really think the activation hook should be triggered at the model level. Is there a reason you're confining it to the view? There might be a situation where an editor is hidden and we could delay package activation, but that does seem pretty rare.